x-pack/filebeat/input/streaming/input_test.go

```diff
@@ -453,7 +460,7 @@ var inputTests = []struct {
 				"wait_max":     "2s",
 			},
 		},
-		wantErr: fmt.Errorf("failed to establish WebSocket connection after 2 attempts with error websocket: bad handshake and (status 403)"),
+		wantErr: fmt.Errorf("failed to establish WebSocket connection after 2 attempts with error websocket: bad handshake"),
```

Comment:
I think only an error should be enough here. The status code can be easily derived from the error if required.